.circleci/config.yml

```diff
@@ -54,6 +54,7 @@ jobs:
               38) pyenv_py_ver="3.8.7" ;;
               39) pyenv_py_ver="3.9.1" ;;
               310) pyenv_py_ver="3.10.1" ;;
+              311) pyenv_py_ver="3.11.0" ;;
```

Comment:
Note that 3.11.1 is out --> https://docs.python.org/release/3.11.1/whatsnew/changelog.html

Reply:
pyenv had 3.11.0 available (via the update command below)